study.md

```diff
@@ -62,21 +74,41 @@ delicious apple.
 Copy the JSON that you wrote for the previous question and include the new information about the kitchen.
 
 ```json
-<!-- your answer here -->
+{
+  "kitchen": {
+    "cupboard": {
+      "bowls and plates"
+    }
+  } "electric stove": {"heating elements": {
+    "small", "medium", "large", "extra large"
+  }
+    "herbs and spice rack": {
+      "chives", "lemon grass", "mint", "cinnamon", "nutmeg", "vanilla"
+    }
+  } "fruit basket": {
+    "apples": [
+      "fuji": 2
+      "gala": 7
+      "golden delicious": 1
+    ]
+  }
+}
 ```
 
 ## JSON Methods
 
 Can you write a method in the JSON that you wrote above?
 
 ```json
-<!-- your answer here -->
+const kitchen = function() {
```

Comment:
Similar to how you cannot write comments in JSON, you actually cannot write methods either. JSON is simply used to represent data!